app/models/concerns/auction_scopes.rb

```diff
@@ -4,6 +4,10 @@ module AuctionScopes
   included do
     scope :accepted, -> { where(status: statuses['accepted']) }
     scope :accepted_pending_payment_url, -> { where(status: statuses['accepted_pending_payment_url']) }
+    scope :accepted_and_pending, lambda {
```

Comment:
I don't **think** this `lambda` is needed (it's also not the style we are using in the rest of this module but I might be missing something)